lib/couchdb/index.js

```
@@ -119,6 +120,9 @@ const findById = R.curry((couch, bucket, projection, id) => R.composeP(
 )(bucket, id))
 
 
+const del = R.curry((couch, bucket, id, rev) => couch.del(bucket, id, rev))
```

this should be deleteById

---

probably something along the lines of this

```javascript
const deleteById = R.curry((couch, bucket, id) =>
  couch.getById(couch, id)
  .then(R.prop('_rev_id')
  .then(couch.deleteByIdAndRevId(couch, bucket, id))
)

---

it should also throw the NotFound error if getById returns nothing.

---

noticed the MemoryStore.deleteById returns true on successful delete. This one should also return true in the same case